docs/cryptodoc/src/05_07_slh_dsa.rst

```diff
@@ -7,6 +7,10 @@ Botan's implementation of the Stateless Hash-Based Digital Signature Standard
 (SLH-DSA) is found in
 :srcref:`src/lib/pubkey/sphincsplus/` and follows [FIPS-205]_.
 
+BSI's [TR-02102-1]_ recommends using SLH-DSA in "hedged" mode with the parameter
+sets that fulfill NIST's category 3 and 5 ("192" or "256" both fast and small).
```

Comment:
both fast and small and both types of hashes?

Reply:
Yes. Adapted to make this more clear.